packages/workers-shared/router-worker/src/index.ts

```diff
@@ -8,6 +10,11 @@ interface Env {
 	CONFIG: RoutingConfig;
 
 	SENTRY_DSN: string;
+	ENVIRONMENT: Environment;
```

Comment:
Have you tested this in local dev when these values won't be populated?

Reply:
Good call! We needed to wrap some of the setData calls to ensure the bindings were present first.